rb/schema/startup.yml

```diff
@@ -0,0 +1,36 @@
+type:       map
+mapping:
+  "title":
+    type:      str
+    required:  yes
+  "mission":
+    type:      str
+    required:  yes
+  "owner":
+    type:      str
+    required:  yes
+  "status":
+    type:      str
+    required:  yes
+  "start":
+    type:      date
+    required:  yes
+  "end":
+    type:      date
+  "incubator":
+    type:      str
+    required:  yes
+    assert:    "['dinsic','mtes','pole-emploi','sgmas'].include?(val)"
+  "link":
+    type:      str
+  "repository":
+    type:      str
+  "stats":
+    type:      bool
+  "contact":
```

Comment:
Je mettrais bien un required là dessus

Reply:
Why not, liste des erreurs si on le rajoute: 
```Running rb/schema/startup.yml on _startup/*.md…
Error in _startup/eac.md: [/] key 'contact:' is required.!
Error in _startup/eac.md: [/contact] value required but none.!
Error in _startup/le-taxi.md: [/] key 'contact:' is required.!
Error in _startup/le-taxi.md: [/contact] value required but none.!
Error in _startup/pass-culture.md: [/] key 'contact:' is required.!
Error in _startup/pass-culture.md: [/contact] value required but none.!
Run on 52 file(s) and found 6 error(s).```